qt/python/mantidqt/mantidqt/widgets/sliceviewer/model.py

```diff
@@ -51,10 +51,10 @@ def __init__(self, ws):
         else:
             raise ValueError("only works for MatrixWorkspace and MDWorkspace")
 
-        wsname = self.get_ws_name()
-        self._rebinned_name = wsname + '_svrebinned'
-        self._xcut_name, self._ycut_name = wsname + '_cut_x', wsname + '_cut_y'
-        self._roi_name = wsname + '_roi'
+        self._ws_name = ws.name()
```

Comment:
Could we keep this as it was and just call `set_ws_name(ws.name())` beforehand? If it uses the `self.get_ws_name()` then that can be the thing that determines how the name is returned and eveything consistently uses the same setter method too.